src/panels/lovelace/common/compute-unused-entities.ts

```diff
@@ -72,11 +72,23 @@ export const computeUnusedEntities = (
   config: LovelaceConfig
 ): string[] => {
   const usedEntities = computeUsedEntities(config);
-  return Object.keys(hass.states)
-    .filter(
-      (entity) =>
-        !usedEntities.has(entity) &&
-        !EXCLUDED_DOMAINS.includes(entity.split(".", 1)[0])
-    )
-    .sort();
+  return [...calcUnusedEntities(hass, usedEntities)].sort();
```

Comment:
I wonder if we actually should convert it to an array here ? Some cases we might not be interested it it being an array (esp as set lookup is always faster)

Reply:
Sets cant be sorted or filter though right?

Reply:
that's up to the consumer to decide how to deal with it?

Reply:
Done